dist/obs-server.spec

```diff
@@ -15,6 +15,7 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 
+%define disable_obs_test_suite 1
```

Comment:
Some testing leftover?

Reply:
Yes, for building faster in my local branch. I'll remove this commit when the package builds without issues. Labelling again as "DO NOT MERGE".